app/main.py

```diff
@@ -1,3 +1,17 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Dict, Any
+
+
+def cache(func: Callable) -> Callable:
+    cache: Dict[Any, Any] = {}
```

Comment:
```suggestion
    cache: Dict[Any, Any] = {}
```
not Any, only hashable objects🙂